src/lib/common/webnative.ts

```diff
@@ -115,3 +117,28 @@ export const loadAccount = async (username: string): Promise<void> => {
     authed: true
   }))
 }
+
+const checkDataRoot = async (username: string): Promise<void> => {
+  let dataRoot = await webnative.dataRoot.lookup(username)
+
+  if (dataRoot) return
+
+  return new Promise((resolve) => {
+    const maxRetries = 20
```

Comment:
@bgins Not familiar with TypeScript module best practices, but feels weird to define this constant here. The name is quite clear! I wondering if it should be moved to the top level of `webnative.ts` and commented? 🤔 

Maybe we wait until we need a retry construct somewhere else?

Now that I'm thinking about it, should this retry behavior be handled by the WebNative library, perhaps with an exponential backoff? (future improvement, not for this PR)